src/components/shared/AnswerChoiceBox.tsx

```diff
@@ -7,10 +7,11 @@ interface AnswerChoiceBoxProps {
   imgSrc?: string;
   text?: string;
   style?: AnswerChoiceBoxStyles;
+  backgroundImg?: string;
```

Comment:
maybe called id instead?
```suggestion
  id?: string;
```